dashboard/dashboard-access.md

```diff
@@ -42,6 +42,8 @@ TiDB Dashboard 可在常见的、更新及时的桌面浏览器中使用，具
 
 登录后，24 小时内将保持自动登录状态。参见[登出](#登出)章节了解如何登出用户。
 
+从 TiDB 5.3 开始，TiDB Dashboard 支持使用任意 SQL 用户登录，参见 [TiDB Dashboard 用户](/dashboard/dashboard-user.md)了解详情。
```

Comment:
As a doc for 5.3, how about modifying L32 to L41 to include this feature? It might be better to give a complete description about a feature in the doc, instead of a description about a partialFeature+improvement.

Reply:
Ok, but we need to commit a new PR to do this, does it?

Reply:
We can introduce a complete user doc for the "non-root" login feature, or do it one by one. I think both are fine. It's up to you.